hw/top_earlgrey/data/top_earlgrey.hjson

```diff
@@ -246,6 +246,13 @@
       reset_connections: {rst_ni: "sys_io", rst_usb_48mhz_ni: "usb"},
       base_addr: "0x40150000",
     },
+    { name: "otbn",
+      type: "otbn",
+      clock_srcs: {clk_i: "main"},
+      clock_group: "trans",
+      reset_connections: {rst_ni: "sys"},
+      base_addr: "0x50000000",
```

Comment:
@eunchan I randomly chose this base address, please let me know if you prefer something else. (I know there has been some work on a global memory map, but I don't remember seeing the final document.)